config.json

```diff
@@ -30,15 +30,15 @@
         "groupId": "androidx.activity",
         "artifactId": "activity",
         "version": "1.3.1",
-        "nugetVersion": "1.3.1.2",
+        "nugetVersion": "1.3.1.3",
```

Comment:
There's about 20 artifacts that are bumping the `nugetVersion` but are not bumping the artifact `version`, What is this for?

Reply:
I don't remember. It was begining of December. I might have rebased/merged some change from other PRs